Doc/reference/compound_stmts.rst

```diff
@@ -788,6 +786,9 @@ An example of a coroutine function::
         do_stuff()
         await some_coroutine()
 
+.. versionchanged:: 3.7
+   ``await`` and ``async`` are now keywords, previously they were only
```

Comment:
nit: please replace the comma with a semicolon.

Reply:
Done!